net-analyzer/prelude-lml/files/prelude-lml.initd

```diff
@@ -0,0 +1,25 @@
+#!/sbin/runscript
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+BIN_LML=/usr/bin/prelude-lml
+PID_LML=/run/prelude-lml/prelude-lml.pid
+
+depend() {
+  need net
+}
+
+start() {
+  ebegin "Starting prelude-manager"
```

Comment:
- s/prelude-manager/prelude-lml/
- Something like `checkpath -d -m 0755 -o root:root /run/prelude-lml` is needed here